website/src/css/pages/_code-insights.scss

```diff
@@ -0,0 +1,92 @@
+.code-insights-page {
+    &__hero-and-header {
+        background-image: url('/code-insights/code-insights-illustration.svg'),
+            url('/code-insights/code-insights-page-bg.svg'),
+            // Fallback while the SVGs are loading
+            radial-gradient(ellipse at bottom right, rgba($blue-mist, 0.5), transparent),
+            radial-gradient(ellipse at bottom left, rgba($light-green2, 0.5), transparent);
+        background-repeat: no-repeat, no-repeat;
+        background-position: center right, top left;
+
+        h4 {
+            font-weight: 400;
+        }
+    }
+
+    /* Icon overrides */
+    .mdi-icon {
+        &.icon {
+            border-radius: 40%;
+            padding: 1rem;
+            background-color: $white;
+            color: $cyan-2;
+        }
+    }
+
+    /* Bootstrap overrides */
+    .tab-header {
+        background-color: transparent !important;
+    }
+
+    .btn-link,
+    .btn-primary {
+        @media only screen and (max-width: 768px) {
+            flex: unset;
+            max-width: unset;
+        }
+    }
+
+    .btn-link {
+        color: $brand-primary-blue;
+        text-decoration: none;
+    }
+
+    .btn-primary {
+        color: $white;
+        background-color: $brand-primary-blue;
+    }
+
+    /* Handle the different code insights chart sizes */
+    .set-height {
+        height: 900px;
+
+        @media only screen and (max-width: 768px) {
+            height: auto;
```

Comment:
It might be a good idea for UX to prevent vertical scroll content jumps when different slides are changed. Or rather, move the arrows above the content (second screenshot) so that you don't notice the jump since the content would be below. But we should stack the arrow/dots UI and the graphs/content up to 990px breakpoint since the text gets pretty crammed between 768–990px (first screenshot).

![Screen Shot 2022-02-18 at 3 00 48 PM](https://user-images.githubusercontent.com/1733936/154753281-3ff2a31b-39ba-458e-8282-ed67ee71046b.png)

Feel free to add input via the suggestion of moving the arrows to the top for better UX (below screenshot) @stephzabala 

![Screen Shot 2022-02-18 at 3 02 00 PM](https://user-images.githubusercontent.com/1733936/154753454-e027e474-eaf6-4e94-808a-4a064aedd2e0.png)